src/rngs/thread.rs

```diff
@@ -90,7 +90,7 @@ thread_local!(
 /// Retrieve the lazily-initialized thread-local random number
 /// generator, seeded by the system. Intended to be used in method
 /// chaining style, e.g. `thread_rng().gen::<i32>()`, or cached locally, e.g.
-/// `let mut rng = thread_rng();`.
+/// `let mut rng = thread_rng();`.  Also called by the `Default` trait.
```

Comment:
I don't think this is very clear. Better to say this is equivalent to `ThreadRng::default()` when using the `Default` trait.

Reply:
updated